parser/src/ast/mod.rs

```diff
@@ -36,6 +39,8 @@ pub struct Source(pub Vec<SourceSection>);
 ///   elements specified. The array length is expected to be a power of 2, but this is not checked
 ///   during parsing.
 /// - RandomValues: Random Values represent the randomness sent by the Verifier.
+/// - EvaluatorFunction: Evaluator Functions take description of the main and auxiliary traces as
+///   input, and returns integrity constraints that are enforced on those trace columns.
```

Comment:
1. nit: there's a typos "take description" -> "take descriptions"
2. I would adjust the end of it to "and enforce integrity constraints on those trace columns."
3. This may need to change if we allow periodic columns or random values as inputs